mail_gateway/models/mail_gateway.py

```diff
@@ -25,12 +25,10 @@ class MailGateway(models.Model):
     )
     webhook_user_id = fields.Many2one(
         "res.users",
-        default=lambda self: self.env.user.id,
+        default=lambda self: self.env.ref("base.user_root"),
```

Comment:
I don't understand why do you change this. What is the problem of the current default of this fields?

Reply:
If the settings are configured by a regular user who does not wish to receive system notifications for every message, it is better not to add them automatically. Additionally, for the webhook user, Odoo follows a similar logic by using the Odoo bot user instead of the current user.